CONTRIBUTING.md

```diff
@@ -38,17 +54,85 @@ OS            | Supported tests
 OS X          | Android, iOS
 Windows       | Android
 
+### Test descriptions
+
+The tests first build the app.
+
+They then check if the required emulators are currently running.
+
+If they are not, then it attempts to start an Android emulator named ```emulator``` (if one exists) and an iOS simulator named ```iPhone 6s (9.3)```. If these emulators don't exist on your computer, the task will exit and the tests will fail.
+
+If you would like the tests to always restart the necessary emulators, add a ```-clean``` to the end of the command you'd like to run.
 
-##### iOS
+The desired unit tests are then run.
 
-To run the iOS tests:
+There is a both a full unit test suite and a "core" set of unit tests that you may run. If you would like to run only the core tests, add a ```-core``` to the end of the command you'd like to run.
```

Comment:
Can we make them run in parallel? I think the only thing stopping us from doing that is the test project paths, which are common, but we should be able to change that. It's not necessary to do it now, but it would be nice to have.